openshift/release/knative-eventing-channelbroker-ci.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       containers:
       - name: broker-controller
         terminationMessagePolicy: FallbackToLogsOnError
-        image: registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-channel-broker
+        image: registry.svc.ci.openshift.org/openshift/knative-v0.15.2:knative-eventing-channel-broker
```

Comment:
why are we dropping these down to `0.15.2` from `0.16.0` ?  Due to deprecation and `0.15.2` release are the last point they were shipped?

Reply:
Good question

I have added some test on the actual commit: https://github.com/openshift/knative-eventing/pull/782/commits/8e2c93c939ded16f48b51ee7f8eeaead132f2f02


BUT.  here is a bit more

for 0.16 the (in 0.15 deprecated) `channelbroker` was deleted:
https://github.com/knative/eventing/pull/3212/files

and for the `replicas: 0` deprecation strategy, Ville did pin the image to the one from the 0.15.0 upstream release, see here:
https://github.com/knative/eventing/blob/release-0.16/config/brokers/channel-broker/deployments/controller.yaml#L38


Now, here we wanna do the same. Since there simply is no real binary for 0.16.0 of the "channel broker", we do pick 0.15.2 since that is the last release of what we have on 0.15.2 (for our Serverless 1.9.0 product)